.github/workflows/release-admin-tools.yml

```
@@ -37,4 +37,5 @@ jobs:
           SRC_REPO: temporaliotest
           DST_REPO: temporalio
           LATEST: ${{ github.event.inputs.latest }}
-        run: go run src/image_copy/main.go
+        working-directory: ./src
+        run: go run image_copy/main.go
```

I always like to do
```suggestion
        run: go run ./image_copy/main.go
```
to be more clear, since go sometimes interprets things as package paths instead of in the filesystem